Sources/Diagnostics/DiagnosticsEvent.swift

```diff
@@ -13,7 +13,7 @@
 
 import Foundation
 
-struct DiagnosticsEvent: Codable, Equatable {
```

Comment:
I wonder if we should keep this here to avoid unnecessary changes... But then again, it does seem unused so I think it should be ok 👍